src/CommonITILValidation.php

```diff
@@ -1360,96 +1444,6 @@ public function getRights($interface = 'central')
         return $values;
     }
 
-
-    /**
-     * Dropdown of validator
-     *
-     * @param $options   array of options
-     *  - name                    : select name
-     *  - id                      : ID of object > 0 Update, < 0 New
-     *  - entity                  : ID of entity
-     *  - right                   : validation rights
-     *  - groups_id               : ID of group validator
-     *  - users_id_validate       : ID of user validator
-     *  - applyto
-     *
-     * @return void Output is printed
-     **/
-    public static function dropdownValidator(array $options = [])
```

Comment:
This one is used in `templates/components/itilobject/fields/global_validation.html.twig`. Thus, method should just be deprecated, to not result in a BC break.

Reply:
I removed the deprecation state and update the validator types. It is actually used in ticket creation form:
![image](https://user-images.githubusercontent.com/33253653/162906134-fe1f50ba-3481-4005-9c53-da5361c42311.png)